starter-code/database.js

```diff
@@ -116,8 +120,9 @@ class Database {
         // Implement the query to delete a product
         // productName is the name of the producto to delete 
         // remeber once it's finish to comment callback('Error deleting product');
+        database.collection('products').deleteOne(productName, callback);
```

Comment:
And here we have a problem, because the object key and the variable have different names, then you need to do:
```
database.collection('products').deleteOne('name' : productName, callback);
```
Your code won't work because product doesn't have a key productName. Ask me if this is not clear.